tests/test_collection.py

```diff
@@ -0,0 +1,16 @@
+from slider import CollectionDB
+
+
+def test_collection_simple():
+    collection_db = CollectionDB.from_path("data/collection.db")
```

Comment:
I don't think this is committed to the repo. There is a directory with some example data which was meant for examples and then later tests. I think it has a replay and a beatmap now, but maybe we could add this collection and constituent beatmaps.

Reply:
You are right - I will have to add that in. Though I prefer to have test data in tests directory rather than with the source code.

Reply:
fixed .gitignore https://github.com/jxu/slider/commit/d25b7ddf97f001c78d9fe72edbc5c057a19b15a5
note osu maps use CLRF newlines

Reply:
You should test it out on your end.